packages/project-utils/bundling/function/buildFunction.js

```diff
@@ -54,4 +73,21 @@ module.exports = options => {
             resolve();
         });
     });
+
+    const handlerFile = fs.readFileSync(options.cwd + "/build/handler.js", {
```

Comment:
Use `path.join` to join paths. This is mainly for compatibility with multiple OSes.